src/MCPClient/lib/clientScripts/rights_from_csv.py

```diff
@@ -42,7 +42,6 @@ class RightCsvReader(object):
 
     metadata_applies_to_type = None
     current_row = None
-    object_basis_act_usage = {}
     rows_processed = 0
```

Comment:
It seems to me that these variables should be in the constructor as well, is there a reason you elected to leave them out with this fix?

Reply:
Thanks @sevein. I had a read, and gave the code a whirl. Looks good to me. No reason to expand the scope :+1:

Reply:
Thanks for taking the time to do that!